strax/storage/mongo.py

```diff
@@ -39,14 +39,15 @@ def _read_chunk(self, backend_key, chunk_info, dtype, compressor):
 
         # Query for the chunk and project the chunk info
         doc = self.db[self.col_name].find_one(
-            {**query, chunk_name: {"$exists": True}},
-            {f"{chunk_name}": 1})
+            {**query, "chunk_i": chunk_name},
```

Comment:
Is there a compelling reason why we carry `chunk_name` rather than its number? Integers index much more cleanly than `chunk_%i`, and I'm not aware of cases when chunks get non-integer identifiers.

Reply:
Changed it, it's also more consistent